server/main.lua

```diff
@@ -393,6 +393,33 @@ QBCore.Commands.Add("paylawyer", Lang:t("commands.paylawyer"), {{name = "id",hel
     end
 end)
 
+QBCore.Commands.Add('fine', 'Fine A Person', {{name = 'id', help = 'Player ID'}, {name = 'amount', help = 'Fine Amount'}}, false, function(source, args)
+    local biller = QBCore.Functions.GetPlayer(source)
+    local Ply = QBCore.Functions.GetPlayer(tonumber(args[1]))
+    local billed = QBCore.Functions.GetPlayer(tonumber(args[1]))
```

Comment:
No reason to use duplicate variables here, delete one of them and update the code accordingly.